appointment.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Veterinary Clinic</title>
+    <link rel="stylesheet" href="assets/css/style.css">
+    <link rel="stylesheet" href="assets/css/calendar.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/js/bootstrap.bundle.min.js"></script>
+    <script src="assets/js/loadcomponents.js"></script>
+
+</head>
+<div id="header-placeholder"></div>
+<main class="container my-4 flex-grow-1">
+
+
+    <div id="calendar-page" >
+        <div class="calendar-header">
+            <button id="prev-month">Previous</button>
+            <div id="current-month"></div>
+            <button id="next-month">Next</button>
+        </div>
+        <div class="calendar">
+            <!-- Dias da semana -->
```

Comment:
```suggestion
```

We need to avoid unnecessary comments. No comments that can be easily derived from code.
Also, no Portuguese.